telemetry/definitions/commonDefinitions.json

```diff
@@ -3844,12 +3976,77 @@
                 {
                     "type": "codeTransformSessionId",
                     "required": true
+                },
+                {
+                    "type": "localBuildSystemVersion",
+                    "required": false
+                }
+            ]
+        },
+
+        {
+            "name": "codeTransform_uploadProject",
```

Comment:
If we're not emitting a metric for API calls, do we need to emit one for this PUT request? I'm indifferent but just wanted to mention it.

Reply:
This metric will be a step in user journey to indicate whether project has been uploaded successfully (including both zipping + upload).